prov/sockets/src/sock_cntr.c

```diff
@@ -420,7 +421,11 @@ static int sock_cntr_verify_attr(struct fi_cntr_attr *attr)
 	switch (attr->wait_obj) {
 	case FI_WAIT_NONE:
 	case FI_WAIT_UNSPEC:
+#ifndef _WIN32
 	case FI_WAIT_MUTEX_COND:
+#else /* _WIN32 */
+	case FI_WAIT_CRITSEC_COND:
+#endif /* _WIN32 */
```

Comment:
*_valid_attr functions are guarded by #ifdef to prevent usage of incorrect object types, implementation of wait itself uses case cond-case crit to simplify logic (actually there is absolutely same logic for unix & windows, just different datatype names)